com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -191,23 +190,89 @@ public UniversalRenderer(UniversalRendererData data) : base(data)
 
             // RenderTexture format depends on camera and pipeline (HDR, non HDR, etc)
             // Samples (MSAA) depend on camera and pipeline
-            m_CameraColorAttachment.Init("_CameraColorTexture");
-            m_CameraDepthAttachment.Init("_CameraDepthAttachment");
-            m_DepthTexture.Init("_CameraDepthTexture");
-            m_NormalsTexture.Init("_CameraNormalsTexture");
-            if (this.renderingMode == RenderingMode.Deferred)
+            m_CameraAttachments.color = RTHandles.Alloc(
+                Vector2.one,
+                depthBufferBits: DepthBits.None,
+                colorFormat: GraphicsFormat.B10G11R11_UFloatPack32, // TODO there's a determine format we can use here
+                filterMode: FilterMode.Bilinear,
+                dimension: TextureDimension.Tex2D,
+                useMipMap: false,
+                autoGenerateMips: false,
+                enableMSAA: false,
+                name: "_CameraColorTexture");
+
+            bool bindMS = false;
+#if ENABLE_VR && ENABLE_XR_MODULE
+            bindMS = !SystemInfo.supportsMultisampleAutoResolve && SystemInfo.supportsMultisampledTextures != 0;
+#endif
+            m_CameraAttachments.depth = RTHandles.Alloc(
+                Vector2.one,
+                depthBufferBits: k_DepthStencilBufferBits,
+                colorFormat: GraphicsFormat.DepthAuto,
+                filterMode: FilterMode.Point,
+                dimension: TextureDimension.Tex2D,
+                useMipMap: false,
+                autoGenerateMips: false,
+                enableMSAA: false,
+                bindTextureMS: bindMS,
+                name: "_CameraDepthAttachment");
+
+            m_DepthTexture = RTHandles.Alloc(
+                Vector2.one,
+                depthBufferBits: DepthBits.Depth32,
+                colorFormat: GraphicsFormat.DepthAuto,
+                filterMode: FilterMode.Point,
+                dimension: TextureDimension.Tex2D,
+                enableMSAA: false,
+                name: "_CameraDepthTexture");
+            m_NormalsTexture = RTHandles.Alloc(URPShaderIDs._CameraNormalsTexture, "_CameraNormalsTexture");
+            if (renderingMode == RenderingMode.Deferred)
```

Comment:
Should create upon demand, ideally in the GBuffer pass like it was with GetTemporaryRT.